templates/courseCategories.phtml

```diff
@@ -26,7 +26,7 @@ require 'partials/layout/nav.phtml';
             <th class="col-xs-9">Category</th>
             <th class="col-xs-3">Action</th>
         </tr>
-        <?php echo Portal\ViewHelpers\CourseCategoriesViewHelper::displayCourseCategories($categories); ?>
+        <?= Portal\ViewHelpers\CourseCategoriesViewHelper::displayCourseCategories($categories); ?>
```

Comment:
It would have been way better to find/replace all these tags in a separate PR, so it would be easier to find the lines that are relevant only to this one!